group/group-id/README.md

```diff
@@ -47,24 +49,24 @@ that to an unsafe (public) context.
 If we're using `group_id` in `recps`, then we need a way to map this to the key we'll use for boxing.
 
 When you're entrusted with access to a private group, you will receive:
+
 - the location of the initial message for that group
 - the `group_key` for that group
 
 This will allow you to derive the `group_id` for that group (using definition above).
 You will need to keep an index which will allow you map `group_id` to `group_key` so that when you see
 a `recps` field containing the cloaked `group_id` you'll know which shared key to include in envelope `recp_keys`
 
-
 ## Design Considerations
 
-1. the parts you could identify a group uniquely by are: 
-    - it's initialisation message
-      - the public id of this message is knowable by all
-      - the `msg_key` for this message is only know to people in the group
-      - the `read_key` is known to anyone who can read the message (in group or been given permission)
-    - it's private `group_key` for decrypting all messages
+1. the parts you could identify a group uniquely by are:
+   - it's initialisation message
+     - the public id of this message is knowable by all
+     - the `msg_key` for this message is only know to people in the group
+     - the `read_key` is known to anyone who can read the message (in group or been given permission)
+   - it's private `group_key` for decrypting all messages
 2. the above defintion uses the more general "cloaked message id" spec, and has the advantages:
-    - it is not derived from the `group_key`, so this could later be cycled (potentially)
-    - anyone who can read the init message can reference the group, without being able to read the whole group history (you only need the read capability for the group init messsage)
+   - it is not derived from the `group_key`, so this could later be cycled (potentially)
+   - anyone who can read the init message can reference the group, without being able to read the whole group history (you only need the read capability for the group init messsage)
 3. this means the `group_id` is bound to the feed
-    - it can't be set / determined until a group initialisation message is published
+   - it can't be set / determined until a group initialisation message is published
```

Comment:
These cosmetic changes are quite distracting from the core of this PR.
Please move such things into other PRs in future <3

Also, I note you've changed it from an even number of spaces indented to an odd number? I would assume that would cause problems? Am I wrong?

Reply:
i assume it's not a problem, i think it's Prettier doing the formatting

Reply:
actually this seems to have changed it from 3 to 4

Reply:
oh no it ruined some embedded code. fixing now

Reply:
fixed